src/main/java/nl/tudelft/ti2206/group9/level/save/Parser.java

```diff
@@ -59,20 +54,14 @@ private Parser() { }
      */
     static void loadGame(final String path) {
         try {
-            final List<String> lines  = new ArrayList<>();
             final URL pathURL = new File(path).toURI().toURL();
             final InputStream stream = pathURL.openStream();
-            final BufferedReader reader = new BufferedReader(
-                    new InputStreamReader(stream, "UTF-8"));
-
-            while (reader.ready()) {
-                lines.add(reader.readLine());
-            }
-            final String mainString = createString(lines);
-            final String decryptedMain = decrypt(mainString);
+            final Base64Reader reader = new Base64Reader(new BufferedReader(
+                    new InputStreamReader(stream, "UTF-8")));
```

Comment:
Wouldn't it be easier to create a factory method in `Base64Reader` (maybe also in Writer) that creates a `Base64Reader` from an InputStream? That would greatly enhance the readability of these lines.

<review, logic, interface